.devcontainer/devcontainer.json

```diff
@@ -35,19 +43,17 @@
       "label": "noVNC"
     }
   },
-
-  // Use 'postCreateCommand' to run commands after the container is created.
-  // "postCreateCommand": "uname -a",
   // Configure tool-specific properties.
   "customizations": {
     "vscode": {
       "extensions": [
         "esbenp.prettier-vscode",
         "mhutchie.git-graph",
-        "ms-python.black-formatter"
+        "ms-python.black-formatter",
+        "ms-python.python",
+        "ms-iot.vscode-ros"
       ]
     }
-  }
-  // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

Comment:
Why? I was used to the standard `vscode` user 😢

Seriously, of running the Dev Container under the `ubuntu` user is a prerequisite ROS, we should check whether the `vscode` user is hard coded in some scripts or `*.md` files